### ISSUE TITLE
grayscale  images, is its loss  normal?

### ISSUE THREAD

**Issue by myw8** [carpedm20/DCGAN-tensorflow#68]
,I use 64*64 grayscale images (not RGB) of Chinese characters, When training after a period of training the g_loss is a lot of bigger than d_loss,  is it  normal?
![920c73b4-c315-11e6-83f1-759d5a86b1d4](https://cloud.githubusercontent.com/assets/17568519/21287724/50728f6c-c4ad-11e6-9c4f-82fdbd334dc3.png)

**Comment by carpedm20** [comment]
You can check the samples that are generated while the training is going on. Usually the `d_loss` and `g_loss` does have different scale *with my code* (this doesn't happen usually with others paper and code I guess). But if the loss doesn't converge or diverge, the samples are pretty good.

**Comment by myw8** [comment]
I am very puzzled, I put the celeba into gray, g_loss converge, and I used the mnist grayscale image test is also normal. Is it the problem of my pictures, the Chinese picture is generated using matlab,It is too strange.
![015fbc34-c317-11e6-8fbc-1b67f61ade30](https://cloud.githubusercontent.com/assets/17568519/21294456/c25ca610-c577-11e6-8254-5073fc0c3213.png)
